src/aap_eda/core/enums.py

```diff
@@ -70,6 +70,7 @@ class ActivationStatus(DjangoStrEnum):
     STARTING = "starting"
     RUNNING = "running"
     PENDING = "pending"
+    PENDING_WORKERS_OFFLINE = "pending (workers offline)"
```

Comment:
I think we need to discuss this. The last agreement was to use a regular "pending" with a different status message. I think a new pending status is redundant and it would require changes in the UI as well.  
I may missing something but I think we don't need a new pending status.